### PR TITLE
Improve right panel resize sensitivity

### PR DESCRIPTION
**Pull request by ymichael**
## Summary

- replace the pseudo-only right-panel grab zone with a real 12px pointer target
- keep the visible zero/1px seam and existing panel layout, constraints, animations, and keyboard behavior
- apply the same hit-target geometry to the split-workspace empty panel
- add regression coverage for both resize paths

## Why

The sidebar uses a physical 12px, high-stacking resize strip. The right panel instead relied on a zero/1px handle plus synthetic margins around a pseudo-element, making pointer acquisition less dependable around adjacent panel content. The new child element owns hit testing while the panel library still recognizes the full region.

## Testing

- pnpm exec turbo run test --filter=@bb/app -- src/components/secondary-panel/ThreadSecondaryPanel.collapseControl.test.tsx
- pnpm exec turbo run test --filter=@bb/app -- src/views/thread-detail/SplitThreadArea.test.tsx
- pnpm exec turbo run typecheck --filter=@bb/app
- pnpm exec turbo run lint --filter=@bb/app (zero errors; existing warnings only)
- git diff --check

### CODE REVIEW

**Comment by SawyerHood** [comment]
🚨 **SLOP COP** 🚨 · `review`

I am the SlopCop. I am reviewing this pull request for security, code quality, performance, architecture, and end-to-end behavior.

The stack contains PR #1091 only. I found no draft or unauthorized stack PRs to skip.

**Comment by SawyerHood** [comment]
🚨 **SLOP COP** 🚨 · `review`

Security phase complete: I found no security issues.

The change adds a transparent pointer target. It does not change authentication, authorization, network, storage, IPC, dependencies, or untrusted-data paths.

**Comment by SawyerHood** [comment]
🚨 **SLOP COP** 🚨 · `review`

Performance phase complete: I found no performance issues.

The change adds one static child to each existing resize handle. It adds no timer, handler, state update, rerender path, or layout write.

File: apps/app/src/components/secondary-panel/ThreadSecondaryPanel.tsx [hunk 1]
```diff
@@ -997,15 +998,15 @@ function SecondaryPanelResizeHandle({
       onDragging={onDragging}
       hitAreaMargins={PANEL_RESIZE_HIT_AREA_MARGINS}
       className={cn(
-        "group relative shrink-0 overflow-visible transition-[width,opacity,background-color] before:absolute before:inset-y-0 before:-left-1.5 before:-right-1.5 before:content-['']",
+        "group relative z-[5] shrink-0 overflow-visible transition-[width,opacity,background-color]",
```
Comment (SawyerHood) [inline apps/app/src/components/secondary-panel/ThreadSecondaryPanel.tsx]: 🚨 `slopcop/review` — The resize target loses half of its width beside a pane header.

This `z-[5]` parent creates a stacking context below the bounded pane header at `z-[21]`. The child target's `z-10` cannot escape it. At the header row, the header covers the six-pixel left overhang. The split divider already uses `z-[25]` for this exact overlap. Please give this handle the same effective layer, and add an interaction test for the header boundary.

**Comment by SawyerHood** [comment]
🚨 **SLOP COP** 🚨 · `review`

Code-quality and architecture phase complete: I found one medium issue and posted it inline.

The handle sits below the pane header, so the header covers half of the new target at the header row. The tests only check markup and classes. They do not test a drag at this overlap.

The PR also duplicates the handle structure in two files. A shared handle component could prevent later drift, but this refactor is not required here.

**Comment by SawyerHood** [comment]
🚨 **SLOP COP** 🚨 · `review`

End-to-end phase complete on the top PR.

I started the PR build and used a real browser. The normal layout exposed all five tested points across the 12-pixel target.

A drag from the outer edge moved 60 pixels and changed the panel width by 59.94 pixels. The bounded pane-header overlap remains the inline finding.

The app test suite passed 2,374 tests in 317 files. The app type check also passed.

**Review by SawyerHood — COMMENTED** [review]
🚨 **SLOP COP** 🚨 · `review`

Review result: one medium finding.

The new 12-pixel target works in the normal layout. However, its `z-[5]` parent keeps it below the `z-[21]` bounded pane header. The header covers the target's left half at that row. I posted the exact issue inline.

Security: no findings.

Performance: no findings.

Architecture: the two handle implementations duplicate structure and state classes. A shared component could prevent drift, but this refactor is not required here.

Validation:

- The real browser drag moved 60 pixels and changed panel width by 59.94 pixels.
- All 2,374 app tests passed across 317 files.
- The app type check passed.

I used only a comment review. I did not approve or request changes.

File: apps/app/src/components/secondary-panel/ThreadSecondaryPanel.tsx (hunk 1 above)
Comment (ymichael) [reply apps/app/src/components/secondary-panel/ThreadSecondaryPanel.tsx]: Addressed in 7edc3c77a. Both right-panel resize handles now use a shared z-[25] layer matching SplitDivider, so the full 12px target stays above z-[21] bounded pane headers. I also added a dedicated split-layout regression that focuses the plugin pane, exposes the adjacent empty-panel handle, and verifies its effective stacking layer is above the pane header. The two focused suites (45 tests total) and the app typecheck pass.
